CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 * Added support for PHP 8.2 [#2136](https://github.com/ruflin/Elastica/pull/2136)
+* Added missing `@throws` annotations [#2152](https://github.com/ruflin/Elastica/pull/2152)
```

Comment:
```suggestion
* Added missing `@throws` annotations to Client::request and related methods [#2152](https://github.com/ruflin/Elastica/pull/2152)
```

Reply:
Nit: Added a bit more details so reading only the changelog should users give an understanding on what changed.